Makefile

```diff
@@ -1,13 +1,11 @@
-VIRTUALENV = virtualenv-2.7
+VIRTUALENV = virtualenv
```

Comment:
> Why would you force 2.7?
because it's the current use case: a script for bob

> I would rather support Python 3.4 as well.
for now we don't have a use case, let's not spend extra time on supporting several python versions